defaults/main.yml

```diff
@@ -1,6 +1,7 @@
 # defaults file
 ---
 nagios_server_htpasswd_users: []
+nagios_server_htpasswd_realm: 'Nagios4'
```

Comment:
Maybe name it `nagios_server_htdigest_realm`?

Reply:
yes, it more accurate. Will modify it accordingly.